apps/web/src/app/[locale]/challenge/_components/editor-shortcuts/editor-shortcuts-form.tsx

```diff
@@ -23,87 +23,67 @@ const Key = (props: PropsWithChildren) => {
     </kbd>
   );
 };
-
+function isMacOS() {
+  return navigator.platform.toLowerCase().includes('mac');
+}
 export function EditorShortcuts() {
+  const cmdOrCtrl = isMacOS() ? 'Cmd' : 'Ctrl';
+  const optionOrAlt = isMacOS() ? 'Option' : 'Alt';
```

Comment:
Probably not worth it because the variable name isn't great (and the perf difference is probably VERY small), but we can avoid calling `isMacOS` two times:
```suggestion
  const isMac = isMacOS()
  const cmdOrCtrl = isMac ? 'Cmd' : 'Ctrl';
  const optionOrAlt = isMac ? 'Option' : 'Alt';
```